tests/filecheck/dialects/riscv/riscv_ops.mlir

```diff
@@ -176,6 +176,17 @@
   // CHECK-NEXT: "riscv.ecall"() : () -> ()
   "riscv.ebreak"() : () -> ()
   // CHECK-NEXT: "riscv.ebreak"() : () -> ()
+  "riscv.directive"() ({ }) {"directive" = ".space", "value" = "1024"} : () -> ()
+  // CHECK-NEXT: "riscv.directive"() ({
+  // CHECK-NEXT: }) {"directive" = ".space", "value" = "1024"} : () -> ()
+  "riscv.directive"() ({
+    "riscv.directive"() ({
+    }) {"directive" = ".space", "value" = "1024"} : () -> ()
+  }) {"directive" = ".bss"} : () -> ()
+  // CHECK-NEXT: "riscv.directive"() ({
+  // CHECK-NEXT:   "riscv.directive"() ({
```

Comment:
```suggestion
  // CHECK-NEXT:   "riscv.directive"() {"directive" = ".space", "value" = "1024"} : () -> ()
```
I think we might be better off with no region here. I think this is the correct syntax, but I might be wrong. There should be examples in other tests